confluent_kafka/error.py

```diff
@@ -20,13 +20,9 @@
 from confluent_kafka.serialization import SerializationError
 
 
-class ConsumeError(KafkaException):
+class _KafkaClientError(KafkaException):
     """
-    Wraps all errors encountered during the consumption of a message.
-
-    Note:
-        In the event of a serialization error the original message contents
-        may be retrieved from the ``message`` attribute.
+    Wraps all errors encountered by a Kafka Client
 
     Args:
         error_code (KafkaError): Error code indicating the type of error.
```

Comment:
This not supposed to be a KafkaError, right?

Reply:
Unfortunately we can't change the name as this will break the API.